test/minikube/minikube_long_restore_test.go

```diff
@@ -333,7 +333,7 @@ func TestKubernetesRestoreWithStorageGroups(t *testing.T) {
 	if os.Getenv("NUODB_LICENSE") != "ENTERPRISE" && os.Getenv("NUODB_LICENSE_CONTENT") == "" {
```

Comment:
Didn't realize we had coverage of SGs in these tests. I guess this was to exercise restore specifically, which was designed to work with SGs.

Reply:
Right. The test was setting up SGs using `nuocmd` and not it's more declarative.